docs/en/compute/Spark-On-Kubernetes.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: global
+title: Deploy Alluxio on Kubernetes
+nickname: Kubernetes
+group: Deploying Alluxio
+priority: 3
+---
+
+Alluxio can be run on kubernetes. This guide demonstrates how to run a Spark job on Alluxio
+running in a kubernetes environment.
+
+* Table of Contents
+{:toc}
+
+## Basic Tutorial
+
+This tutorial walks through an example Spark job on Alluxio in kubernetes.
+
+### Prerequisites
+
+- A kubernetes cluster (version >= 1.8).
+
+### Download Binaries
+
+Download the desired Spark version.
+```bash
+wget http://apache.mirrors.tds.net/spark/spark-2.4.0/spark-2.4.0-bin-hadoop2.7.tgz
+tar -xf spark-2.4.0-bin-hadoop2.7.tgz
+cd spark-2.4.0-bin-hadoop2.7
+```
+
+If running the count example, download the Alluxio examples jar.
```

Comment:
the reader doesn't know anything about the count example yet at this point in the doc - can we introduce the idea from that perspective?